src/components/organisms/news/news.html

```diff
@@ -0,0 +1,50 @@
+<div class="news">
```

Comment:
```suggestion
<div class="news">
  <div class="news-and-events-title">
    <h2>News<span class="caption">お知らせ</span></h2>
  </div>
  <div class="list">
　．．．
　．．．
  </div>
  <a href="#" class="news-and-events-more"
    >お知らせ一覧<i class="fas fa-chevron-right"></i
   ></a>
</div>         
```
現在.listクラスのついたdivタグの中にnews-and-events-moreをいれていると思うのですが
これを以上のように.listクラスと同じ階層にnews-and-events-more置くのどうでしょうか。
その後、cssで　.news{}　にdisplay:flex;とalign-self: flex-end;などを使って右に位置調整できないでしょうか